src/styles/less/InstructionsModal.less

```diff
@@ -11,14 +11,58 @@
     padding: 5%;
     min-width: 100%;
 
-    p {
-      font-size: 2.5rem;
-    }
     .accept-button {
       .orangeButtonStyle();
       position: absolute;
       right: 8%;
       top: 82.33%;
     }
+    .ant-modal-body {
+      p {
+        font-size: 3rem;
+      }
+    }
   }
 }
+
+@media @tablet {
+  .instructions-modal {
+    .ant-modal-content {
+        .accept-button {
```

Comment:
I see you nested your css selectors - not that this is an issue at all but the rest of the css (at least the stuff I was looking at) is written un-nested. Is there a difference in performance doing it one way over the other?

![Screen Shot 2021-04-21 at 3 22 05 PM](https://user-images.githubusercontent.com/62610548/115628340-5b54da80-a2b5-11eb-9659-b429f4921013.png)

Reply:
Hey Rob, that's a good point. The reason I nested the selectors is due to my understanding of that being the best practice with LESS. In addition to that, the rest of the LESS file was written with a similar pattern. If I am mistaken about that being the best practice, please let me know!..

Reply:
Cool - Yes, personally I have always nested my CSS as I think it is more readable that way.

Reply:
Just to clarify:

The nesting with LESS is a way to apply styles only to a parent's children.
For example, the code snippet you commented on will only apply styles to `.accept-button` classes that are children of elements with the class `.ant-modal-content`